ui/demo/less/reader.less

```diff
@@ -72,3 +72,16 @@ body {
     font-weight: 300;
   }
 }
+
+.reader__outline-item {
+  > a {
+    transition: filter 200ms ease-in-out;
+  }
+  &.reader__outline-item--target-visible {
+    > a {
+      filter: hue-rotate(45deg);
+      transition-duration: 400ms;
+    }
+  }
+}
```

Comment:
What these rules for (sorry I couldn't get it from your vid)? Also I think we try to avoid defining rules with html elements. It would be great if you can create css classes for `a`

Reply:
@yensung i think it just those delay that we saw from Paul demo last time for the demo app.

Reply:
@yensung i realized there is no need for this so i removed them